include/functions.h

```diff
@@ -1318,7 +1318,7 @@ s16 Camera_ClearFlags(Camera* camera, s16 flags);
 // void func_800DFFAC(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE2 param_5, UNK_TYPE2 param_6, UNK_TYPE2 param_7);
 // UNK_TYPE4 func_800E007C(Camera* param_1, Camera* param_2);
 // UNK_TYPE4 func_800E01AC(void);
-// void func_800E01B8(void);
+Vec3f* Camera_GetSkyboxOffset(Vec3f* dst, Camera* camera);
```

Comment:
```suggestion
Vec3f* Camera_GetQuakeOffset(Vec3f* dst, Camera* camera);
```
We found out from OoT that `Camera_GetSkyboxOffset` is wrong and that the real name should actually be `Camera_GetQuakeOffset`

Reply:
Acknowledged and modified accordingly.